protoc-gen-gogo/protoc-gen-combo/combo.go

```diff
@@ -70,8 +73,22 @@ func (this MixMatch) Gen(folder string, news []string) {
 	var regenerate = exec.Command("protoc", args...)
 	out, err := regenerate.CombinedOutput()
 	if err != nil {
-		fmt.Printf("%s\n", string(out))
-		panic(err)
+		fmt.Print("protoc-gen-combo: error:", err)
```

Comment:
The out variable provides more information than the err variable, when an error occurs.